superset-frontend/src/SqlLab/components/SouthPane/index.tsx

```diff
@@ -152,7 +154,7 @@ export default function SouthPane({
             query={latestQuery}
             actions={actions}
             user={user}
-            height={innerTabContentHeight}
+            height={innerTabContentHeight + EXTRA_HEIGHT_RESULTS}
```

Comment:
@evans we should add a story to rework these components, as they should use flexbox instead of manually calculating heights

Reply:
Feel free to add one into the backlog. I seem to recall looking at this before, but there was some sort of wrinkle in implementing the library used to allow draggable resizing of the top/bottom panes.

Reply:
oh yeah, good call. We can take a look down the road